<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EffectsGen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="EffectsGen.qrc">
    <normaloff>:/EffectsGen/EffectsGen.png</normaloff>:/EffectsGen/EffectsGen.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="Track" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSlider" name="hs_StartTime">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_StartTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="hs_EndTime">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_EndTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QComboBox" name="cb_Models"/>
    </item>
    <item>
     <widget class="QComboBox" name="cb_Effects"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Layer:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_Layer">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Length:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_Length">
        <property name="suffix">
         <string>ms</string>
        </property>
        <property name="minimum">
         <number>25</number>
        </property>
        <property name="maximum">
         <number>10000000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Gap:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_Gap">
        <property name="suffix">
         <string>ms</string>
        </property>
        <property name="maximum">
         <number>10000000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tw_Properties">
      <property name="columnCount">
       <number>2</number>
      </property>
      <column/>
      <column/>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_add">
      <property name="text">
       <string>Add</string>
      </property>
      <property name="icon">
       <iconset resource="EffectsGen.qrc">
        <normaloff>:/EffectsGen/icons/add.png</normaloff>:/EffectsGen/icons/add.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect_xLights"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionOpen_Logs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="EffectsGen.qrc">
     <normaloff>:/EffectsGen/icons/cancel.png</normaloff>:/EffectsGen/icons/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionOpen_Logs">
   <property name="icon">
    <iconset resource="EffectsGen.qrc">
     <normaloff>:/EffectsGen/icons/book_open.png</normaloff>:/EffectsGen/icons/book_open.png</iconset>
   </property>
   <property name="text">
    <string>Open Logs...</string>
   </property>
  </action>
  <action name="actionConnect_xLights">
   <property name="icon">
    <iconset resource="EffectsGen.qrc">
     <normaloff>:/EffectsGen/icons/connect.png</normaloff>:/EffectsGen/icons/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect to xLights</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="EffectsGen.qrc">
     <normaloff>:/EffectsGen/EffectsGen.png</normaloff>:/EffectsGen/EffectsGen.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Track</class>
   <extends>QWidget</extends>
   <header>Track.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="EffectsGen.qrc"/>
 </resources>
 <connections/>
</ui>
